kaldb/src/main/java/com/slack/kaldb/chunkManager/IndexingChunkManager.java

```diff
@@ -208,12 +209,14 @@ private void doRollover(ReadWriteChunkImpl<T> currentChunk) {
             @Override
             public void onSuccess(Boolean success) {
               if (success == null || !success) {
+                LOG.warn("roll over failed.");
```

Comment:
We should use a consistent approach on these log messages to make them more readable - most currently are sentence cased without punctuation. I'm not particular about which format as long as we are consistent with one.